src/Materials/effect.ts

```diff
@@ -911,6 +950,15 @@ export class Effect implements IDisposable {
         this._engine.bindUniformBlock(this._pipelineContext!, blockName, index);
     }
 
+    /**

```

Comment:
Same here, could be great to move in a webgpu extension

Reply:
Code moved.